package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.3",
   "private": true,
   "dependencies": {
+    "eslint": "^5.16.0",
     "moment": "2.19.3",
     "node-sass-chokidar": "1.2.2",
-    "npm": "^6.2.0",
-    "npm-run-all": "^4.1.2",
-    "prop-types": "^15.6.0",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "react-router": "^4.2.0",
-    "react-router-dom": "^4.2.2",
+    "npm": "^6.14.9",
```

Comment:
I'm looking at this… trying to work out _why_ `npm` is a dependancy here… Have you got any ideas?!